data/human/campaign events.txt

```diff
@@ -155,11 +155,87 @@ mission "event: war begins"
+			choice
+				`	"Of course."`
+					goto summary
+				`	"What's happening?"`
+				`	"What makes you think you can just order me to leave?"`
+					goto noncompliance
+
+			`	The liaison officer sighs. "Captain, I'm afraid the planet is in a state of emergency. Yesterday, some rogue planets down south decided they wanted to secede from the Republic. Sent Parliament an ultimatum, wanted electoral and tax reform, free trade, all the same things they've been complaining about for years. No big deal. Three hours ago, though, someone - quite possibly this same 'Free Worlds' group - dropped nuclear bombs the Navy Yard on Geminus and on the Galactic Stock Exchange here on Martini. At least, we presume that was the target, since the bomb took about a quarter of the city with it, as well as the planet's main hospital. We've got about a hundred thousand injured people, another four hundred thousand to screen and treat for radiation poisoning, and another million people to evacuate."`
+			choice
+				`	"I'll get out of the way, then."`
+					goto end
+				`	"Sorry, I didn't know. I'll be out of your hair as soon as I can lift off."`
+					goto end
+				`	"Is there any way I can help?"`
+					goto help
+
+			label noncompliance
+			`	"Look, captain. The government here on <planet> has declared a state of emergency, and that means that the spaceport is under martial law. I'm not sure how you managed to get landing clearance, but you shouldn't be here. We've got thousands of injured people to get off-planet, even if we ignore the potential for a major follow-up attack. If you refuse to comply, I will have you arrested by the military police, and your ship will be impounded. If your crew refuses to accept the change in captainship, they will be replaced by troops from the Naval Reserve Command. Think what you will, but we've got human lives to save here and we aren't going to stop just because some hotshot captain is feeling full of themself. Is that clear?"`
+			choice
+				`	"Yes, that's clear."`
+					goto summary
+				`	"Sorry, I didn't know. You won't have any trouble from me."`
+					goto summary
+				`	"Yeah, right. No way are you going to seize my ship."`
+			
+			`	The liaison officer glowers at you, then turns military police officers in full combat gear materialize behind you, grabbing you by the arms and handcuffing you tightly. As you're frogmarched toward a waiting bunker, you watch as the <ship> lifts off without you, leaving you stranded here. In the bunker, one of the soldiers who arrested you frisks you, seizing your pilot's license and handing it to a sergeant who stamps the words "Permanently Revoked" across it. With those two words, you are barred from space forever. It has been less than a year since you bought your ship, and already you have lost the right to use it as the galaxy falls apart on you.`
```

Comment:
```suggestion
			`	The liaison officer glowers at you, then turns. Military police officers in full combat gear materialize behind you, grabbing you by the arms and handcuffing you tightly. As you're frogmarched toward a waiting bunker, you watch as the <ship> lifts off without you, leaving you stranded here. In the bunker, one of the soldiers who arrested you frisks you, seizing your pilot's license and handing it to a sergeant who stamps the words "Permanently Revoked" across it. With those two words, you are barred from space forever. It has been less than a year since you bought your ship, and already you have lost the right to use it as the galaxy falls apart on you.`
```
Interesting!

Reply:
Whoops, I think I missed about half a sentence here.